float32 fuel_level
uint64 timestamp 
uint64 source_time
